glfw/src/errors.zig

```diff
@@ -4,13 +4,25 @@ const c = @import("c.zig").c;
 
 /// Errors that GLFW can produce.
 pub const Error = error{
+    /// GLFW has not been initialized.
+    ///
+    /// This occurs if a GLFW function was called that must not be called unless the library is
+    /// initialized.
+    NotInitialized,
```

Comment:
Did you intend to bring this back? I thought that we'd eliminated the need for NotInitialized and InvalidEnum errors

Reply:
Ahh, is it just so that `convertError` can return the error so we can `unreachable` it later inside the function for more explicit handling? that makes sense

Reply:
Ah, yep, I should've read PR description first - my bad.